Automattic/wp-calypso
# Clean-up the unused flows and steps data from the deprecated Link in Bio code.

#### Proposed Changes

This PR is a follow-up cleaning-up for the deprecated Link in Bio code, as pointed out by @aneeshd16 in the following two comments:

* https://github.com/Automattic/wp-calypso/pull/71512#pullrequestreview-1233505743
* https://github.com/Automattic/wp-calypso/pull/71515#discussion_r1059841456

#### Testing Instructions

1. Go through the `/start` flow, both when logged-out and logged-in to confirm that the updated user step removal code works.
2. Go through `/setup/link-in-bio` and `/setup-link-in-bio-tld`. They should both work unaffected.




#### Pre-merge Checklist





Related to Automattic/martech#1348
